lang/es.yml

```
@@ -0,0 +1,103 @@
+MAIN_MENU: "Menú principal"
+WELCOME: "Bienvenido a {0}"
+APP_NAME: "Legendario Fácil"
+UNCOMPILED_VERSION_DETECTED: "Se detectó una versión decompilada de {0}."
+UNSTABLE_BUILD_WARNING: "Este artifacto puede llegar a ser inestable."
+SELECT_VALID_OPTION: "¡Selecciona una opción válida!"
+EMPTY_PLACEHOLDER: "¡Nada por aquí!"
+SELECT_ACTION: "Selecciona qué quieres hacer:"
+
+ACTIONS:
+    START: "iniciar"
+    IMPORT: "importar"
+    IMPORT_GAME_IN_PATH: 'importar el juego "{0}", localizado en "{1}"'

```

```suggestion
    IMPORT_GAME_IN_PATH: 'importar el juego "{0}", ubicado en "{1}"'
```
Sounds more casual and it's used more frecuently than "localizado".